Sources/ArcGISToolkit/Resources/es.lproj/Localizable.strings

```diff
@@ -6,7 +6,7 @@ content on a remote host. The variable is the host that prompted the challenge.
 "A certificate is required to access content on %@." = "Se requiere un certificado para acceder a contenido en %@.";
 
 /* A button to add new utility trace starting points. */
-"Add New" = "á_Add New_______________Ó";
+"Add New" = "Agregar nuevo";
```

Comment:
@jona7150 We changed some of these strings to follow [Title-style capitalization](https://developer.apple.com/design/human-interface-guidelines/buttons#Content:~:text=Using%20title%2Dstyle%20capitalization%2C%20consider%20starting%20the%20label%20with%20a%20verb%20to%20help%20convey%20the%20button%E2%80%99s%20action%20%E2%80%94%20for%20example%2C%20a%20button%20that%20lets%20people%20add%20items%20to%20their%20shopping%20cart%20might%20use%20the%20label%20%E2%80%9CAdd%20to%20Cart.%E2%80%9D), I'd think this would apply to other languages as well but I'm not sure. Thoughts?